qiskit_nature/drivers/bosonic_bases/__init__.py

```diff
@@ -32,5 +32,16 @@
 
 from .bosonic_basis import BosonicBasis
 from .harmonic_basis import HarmonicBasis
+from ...deprecation import warn_deprecated, DeprecatedType, NatureDeprecationWarning
+
+warn_deprecated(
```

Comment:
Actually, these should all be already deprecated, since they were succeeded by the bases in the properties framework.
If they were not, I don't think its a big problem since the only place to use them was with the already deprecated `WatsonHamiltonian`

The reason that `qiskit_nature.second_q.drivers.bosonic_bases` exists, is because we left the removal of deprecated code to #748